### ISSUE TITLE
UI Bug At the footer

### ISSUE THREAD

**Issue by Ranamalsingh12** [fossology/FOSSologyUI#266]
### UI Bug

I was checking FossologyUI when i found this bug in footer where the version, branch, build showing [] and commit showing : [#Undefined].

#### Screenshots
![Screenshot from 2022-10-05 20-34-41](https://user-images.githubusercontent.com/90625605/194095558-2e0f765b-efa9-49aa-ad88-310434532d6b.png)

Also Can i work on this issue?

**Comment by hritik2002** [comment]
@Ranamalsingh12 are you working on the issue?

**Comment by Ranamalsingh12** [comment]
Hey @hritik2002 i am working on it currently as you know diwali is going on so i am at currently not working but will start working soon

**Comment by hritik2002** [comment]
cool.

**Comment by danger-ahead** [comment]
@Ranamalsingh12 please assign me to this. Also I checked the api. Looks like there's an error with the API. Can you please guide me through this?

**Comment by Ranamalsingh12** [comment]
Hey Shourya i have resolved this issue there is nothing error with the API
they made it that way after my PR will accepted this issue will be resolved

**Comment by hussain-shariff** [comment]
@Ranamalsingh12 is this issue closed?

**Comment by Ranamalsingh12** [comment]
Yes it's closed I made the PR it's going to merge and this issue will be
closed

**Comment by sahilsekr42** [comment]
this issue is still present what happened to the PR

**Comment by sahilsekr42** [comment]
I believe instead of using "unknown " for comparision undefined had to be used

**Comment by sahilsekr42** [comment]
@GMishx @Shruti3004 @shaheemazmalmmd  would love to see the PR being merged if it doesn't cause issues . Also I  would like to contribute more , hence am open to suggestions....

**Comment by Kuldeepagrahari** [comment]
please assign me this issue for hactoberfest contribution
